addons/mass_mailing/static/src/js/mass_mailing_editor.js

```diff
@@ -143,6 +143,31 @@ $.summernote.eventHandler.modules.popover.update = function ($popover, oStyle, i
 ajax.loadXML("/mass_mailing/static/src/xml/mass_mailing.xml", core.qweb);
 
 snippets_editor.Class.include({
+    /**
+     * Add padding in Basic theme(Plain text) template for proper preview
+     *
+     * @override
+     */
+    start: function () {
+        var $basicTheme = this.$editable.find('.o_layout.o_basic_theme');
+        if ($basicTheme.length) {
+            $basicTheme.css('padding', '25px');
+        }
+        return this._super.apply(this, arguments);
+    },
+    /**
+     * Basic theme(Plain text) template should look proper in gmail without padding
+     * So, padding removed when save template
+     *
+     * @override
+     */
+    cleanForSave: function () {
+        this._super.apply(this, arguments);
+        var $basicTheme = this.$editable.find('.o_layout.o_basic_theme');
+        if ($basicTheme.length) {
+            $basicTheme.css('padding', '0px');
+        }
+    },
```

Comment:
Why is this needed in JS ?

Reply:
As per the spec we also have to maintain the padding of text in gmail. Please look at [https://drive.google.com/file/d/0B1uIL9E_zXrrb3dCZVpZcEIwU3c/view](url)
If we remove padding from template then not getting proper preview of template, i mean all texts are placing at left corner like [https://drive.google.com/file/d/1o4mZ4IRlzmwi4ZJybPLPtoYlVg4McCNX/view?usp=drivesdk](url)
That's why we've removed padding on save and reset it in edit mode to maintain preview of template as well as in gmail.

Reply:
I think that what we need to do is:
- Removing the padding of the `.o_layout` element (content element)
- Adding padding to the `#editable_area` element  (editor UI element)

Both in LESS (`theme_default.less` and `mass_mailing.ui.less` I suppose). Should not it solve the problem without any JS ?

Reply:
It solve the problem but not proper preview of template. see [https://drive.google.com/file/d/1JoGKPP8c_omT89QA6Vqd6FaACWX2LzIv/view?usp=drivesdk](https://drive.google.com/file/d/1JoGKPP8c_omT89QA6Vqd6FaACWX2LzIv/view?usp=drivesdk)
is it ok?

Reply:
Well can't we make the `#editable_area` element white as well (instead of explicitly transparent I think)?

Reply:
yes...we can